plan/planbuilder.go

```diff
@@ -325,6 +329,88 @@ func (b *planBuilder) buildAdmin(as *ast.AdminStmt) Plan {
 	return p
 }
 
+func detachIndexColumns(tn *ast.TableName) (indexOffset []int, columnOffset []int) {
```

Comment:
First, I will only detach the first columns in each index columns, which I will not consider   
b as an index column, so I will not use the same algo as a for b.
Second, why should I build a histogram for tuple (a,b)? All the queries is all about a single column if I'm not wrong.